### ISSUE TITLE
Does not open page unless logseq is already running

### ISSUE THREAD

**Issue by Slabhub** [freder/logseq-plugin-open-page-in-logseq#1]

**Comment by freder** [comment]
I currently don't have access to a windows machine; only tested on macos so far.
if you have more insight in how things work on windows, PRs are welcome ;)

**Comment by freder** [comment]
as for what exactly isn't working: does it open a file save dialog for you at all? or is it just the generated .url that doesn't open logseq?

**Comment by Slabhub** [comment]
toolbar icon clicked and nothing happened :) 
i dont know if plugin create url file on any folder

**Comment by freder** [comment]
could you please test if [v0.1.2](https://github.com/freder/logseq-plugin-open-page-in-logseq/releases/tag/v0.1.2) changes anything for you?

**Comment by Slabhub** [comment]
i tested on v0,1.2 . now it works now. 
But url shortcut can not properly unless logseq already opened

**Comment by freder** [comment]
can confirm that this is also happening on macos. opened https://github.com/logseq/logseq/issues/9562

**Comment by Slabhub** [comment]
> can confirm that this is also happening on macos. opened [logseq/logseq#9562](https://github.com/logseq/logseq/issues/9562)

yes i do. on mac , it is working now very well unless logseq is not closed.
